tests/test_distributions.py

```diff
@@ -260,10 +260,15 @@
             "sample": np.array([[1.0, 1.0], [0.0, 1.0]], dtype="float32"),
         },
     },
-    # "Weibull": {
-    #     "scalar_parameters": {"beta": 0.1, "alpha": 1.0},
-    #     "multidim_parameters": {"beta": 0.1, "alpha": 1.0},
-    # },
+    "Weibull": {
+        "scalar_parameters": {"low": 0.0, "high": 1.0, "concentration": 1.0, "scale": 1.0},
+        "multidim_parameters": {
+            "low": np.array([0.0, 0.0], dtype="float32"),
```

Comment:
As above, you have to drop the low and high parameters

Reply:
Done :+1: